src/operator/windowToggle.ts

```diff
@@ -151,7 +157,11 @@ class WindowToggleSubscriber<T, O> extends OuterSubscriber<T, any> {
     }
   }
 
-  closeWindow(index: number) {
+  private closeWindow(index: number): void {
+    if (index < 0) {
```

Comment:
I think it's more idiomatic to check `index === -1` here. Will this ever be less than `-1`?